src/components/MapViewButton.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+    <div id="eegeo-tilt-button-container" class="eegeo-tilt-button-container">
+        <button class="eegeo-ripple-button eegeo-tilt-button" v-on:click="toggleView">{{this.mapViewText}}</button>
```

Comment:
Are these classes from wrld.css?

Reply:
They're supposed to be but they also weren't finding them from the stylesheet? So I just copy and pasted the styles into the bottom style tags.

Reply:
yeah let's investigate this too. Instead of just including it.